common/reviews/api/rush-lib.api.md

```diff
@@ -72,16 +72,14 @@ export class BuildCacheConfiguration {
 // @public
 export enum BumpType {
     // (undocumented)
-    'major' = 5,
+    'major' = 4,
     // (undocumented)
-    'minor' = 4,
+    'minor' = 3,
```

Comment:
We shouldn't change the values of the remaining members. Can you explicitly assign them?

Reply:
This should match the enum in https://github.com/microsoft/rushstack/pull/4157/files#diff-bf074f93daa3215e9e13f1df19918d8f56dc43f671f7e3920531845d646bc93e (src/api/VersionPolicy.ts) - correct?

Removing `preminor` from the enum is going to renumber the enum too - isn't it? In which case do I need to add explicit values to that enum in the source to preserve the values in both locations?

Is worth commenting out (in VersionPolicy.ts) the `preminor` and adding a comment so that it's clear what has happened ? Or is the commit message enough?